include/alpaka/kernel/TaskKernelCpuOmp2Blocks.hpp

```diff
@@ -156,15 +169,17 @@ namespace alpaka
                 *static_cast<WorkDivMembers<TDim, TIdx> const*>(this),
                 blockSharedMemDynSizeBytes);
 
-            // NOTE: schedule(static) does not improve performance.
+            auto oldSchedule = omp::getSchedule();
+            omp::setSchedule(schedule);
```

Comment:
Is this function and `omp::setSchedule` defined for omp < 3.0?

Reply:
That is my function. And it was introduced exactly for the reason to hide checks like if we have OpenMP and OpenMP is at least 3.0